frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -83,13 +86,48 @@ async function deleteUserRoleAssignment(
 
         <div class="dashboard-background-layout">
             <NotificationStack />
-            <UserDataTable
-                :isApplicationSelected="isApplicationSelected"
-                :loading="LoadingState.isLoading.value"
-                :userRoleAssignments="userRoleAssignments || []"
-                :selectedApplicationDisplayText="selectedApplicationDisplayText"
-                @deleteUserRoleAssignment="deleteUserRoleAssignment"
-            />
+            <TablePlaceholder v-if="!isApplicationSelected" />
+            <TabView
+                v-else
+                :pt="{
+                    root: {
+                        style: 'margin-top: 1.5rem',
+                    },
+                    nav: {
+                        style: 'margin: 0.112rem 2.5rem 0;',
+                    },
+                    panelContainer: {
+                        style: 'margin-top: 2.62rem;',
+                    },
+                }"
+            >
+                <TabPanel header="Users">
+                    <template #header>
+                        <Icon
+                            icon="user"
+                            :size="IconSize.small"
+                        />
+                    </template>
+                    <UserDataTable
+                        :isApplicationSelected="isApplicationSelected"
+                        :loading="LoadingState.isLoading.value"
+                        :userRoleAssignments="userRoleAssignments || []"
+                        :selectedApplicationDisplayText="selectedApplicationDisplayText"
+                        @deleteUserRoleAssignment="deleteUserRoleAssignment"
+                    />
+                </TabPanel>
+                <TabPanel
```

Comment:
And let's comment it out for now. No need to display on the page yet.

Reply:
sure, leave the code commented out